Src/FluentAssertions/Primitives/GuidAssertions.cs

```diff
@@ -171,5 +171,5 @@ public AndConstraint<TAssertions> NotBe(Guid unexpected, string because = "", pa
 
     /// <inheritdoc/>
     public override bool Equals(object obj) =>
-        throw new NotSupportedException("Calling Equals on Assertion classes is not supported.");
+        throw new NotSupportedException("Equals is not part of Fluent Assertions. Did you mean Be() instead?");
```

Comment:
🤔 I would also include `BeOneOf`.

Reply:
Will add.  Curious why?  Should I do for enum too?

Reply:
> Should I do for enum too?

Yeah, that's actually a good idea.